src/main/react/features/person/table/PersonTable.tsx

```diff
@@ -12,23 +12,33 @@ import {
   TablePagination,
   TableRow,
   TextField,
+  Typography,
 } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
+
 import { PersonTableHead } from "./PersonTableHead";
 import { Person, PersonClient } from "../../../clients/PersonClient";
 import { PersonDialog } from "../PersonDialog";
 import { CheckBox } from "@material-ui/icons";
 import AddIcon from "@material-ui/icons/Add";
 import Button from "@material-ui/core/Button";
 
-const useStyles = makeStyles({
+const useStyles = makeStyles((theme: any) => ({
   tblEmail: {
     minWidth: 200,
   },
   tblName: {
     minWidth: 170,
   },
-});
+  tblRow: {
+    position: "relative",
+  },
+  link: {
+    color: "black",
+    textDecoration: "none",
+    ...theme?.props?.link,
```

Comment:
@sjorsdev hier gebruik ik dus de `link` class die ik heb gedefineerd in `theme.js`, op deze manier kan ik dit stukje dus hebruiken.